src/components/Footer/Footer.stories.tsx

```diff
@@ -0,0 +1,287 @@
+/*  eslint-disable jsx-a11y/anchor-is-valid */
+import React from 'react'
+
+import { Address } from './Address/Address'
+import { Button } from '../Button/Button'
+import { Footer } from './Footer'
+import { FooterNav } from './FooterNav/FooterNav'
+import { Logo } from './Logo/Logo'
+import { SignUpForm } from './SignUpForm/SignUpForm'
+import { SocialLinks } from './SocialLinks/SocialLinks'
+
+export default {
+  title: 'Footer/Footer',
+  parameters: {
+    info: `
+  USWDS 2.0 Footer component
+  
+  Source: https://designsystem.digital.gov/components/form-controls/#footer
+  `,
+  },
+}
+
+const mockSubmit = (): void => {
+  /* mock submit fn */
+}
+
+const returnToTop = (
+  <div className="grid-container usa-footer__return-to-top">
+    <Button type="button" unstyled>
+      Return to top
+    </Button>
+  </div>
+)
+
+export const SlimFooter = (): React.ReactElement => (
+  <Footer
+    slim
+    returnToTop={returnToTop}
+    primary={
+      <div className="usa-footer__primary-container grid-row">
+        <div className="mobile-lg:grid-col-8">
+          <FooterNav
+            slim
+            links={Array(4).fill(
+              <a className="usa-footer__primary-link" href="#">
+                Primary Link
+              </a>
+            )}
+          />
+        </div>
+        <div className="tablet:grid-col-4">
+          <Address
+            slim
+            items={[
+              <a key="telephone" href="tel:1-800-555-5555">
+                (800) CALL-GOVT
+              </a>,
+              <a key="email" href="mailto:info@agency.gov">
+                info@agency.gov
+              </a>,
+            ]}
+          />
+        </div>
+      </div>
+    }
+    secondary={
+      <Logo
+        slim
+        image={
+          <img
+            className="usa-footer__logo-img"
+            src="/logo-img.png"
```

Comment:
Soooo I realized that I was manually copying over image files we need and loading static files ... just to handle this default logo image.  Now I see there's another pattern in the app of just importing them from `uswds`.
 
like this: `// assets
import closeImg from 'uswds/src/img/close.svg'`. Should I remove my code and just go with that???

Reply:
oh, oops. yes, lol